practice-app/app/game_information_api.py

```diff
@@ -0,0 +1,89 @@
+import requests
+from flask import render_template, request, redirect
+from sqlalchemy import Column, String, Integer, ForeignKey, PrimaryKeyConstraint
+from sqlalchemy.orm import Mapped, mapped_column
+from flask_login import login_required, current_user
+
+from .views import app, Base, session
+
+class FavoriteGame(Base):
+    __tablename__ = 'FavoriteGameTable'
+    id = Column(Integer, autoincrement=True)
```

Comment:
id fields are null in the db, ~~fix required~~ what do you think of it?

Reply:
![image](https://user-images.githubusercontent.com/56600661/236708438-9214f485-3488-4bbe-805f-3d12bc68ea33.png)

Reply:
Fixed now.